rdrf/registry/groups/migrations/0021_alter_workinggroup_options.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 3.2.18 on 2023-05-26 11:34
```

Comment:
Don't forget to use successive migration numbers in the file names when you merge this and #584